packages/dappkit/src/index.ts

```diff
@@ -135,19 +128,22 @@ export interface TxParams<T> {
 }
 
 export async function requestTxSig<T>(
-  web3: Web3,
+  kit: ContractKit,
```

Comment:
We already had this discussion...
"static function which receive kit every time" vs "dappkit obj created once that has a kit instance"

I favor the second, that will also allow you to hide `contractKit` if you want.

Reply:
True, do you mind if we tackle that in a different PR?

Reply:
not at all